UNITTESTS/CMakeLists.txt

```
@@ -146,11 +146,11 @@ set(unittest-includes-base
   "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework/device"
   "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework"
   "${PROJECT_SOURCE_DIR}/../connectivity/cellular/include/cellular/framework/common"
-  "${PROJECT_SOURCE_DIR}/../features/lorawan"
-  "${PROJECT_SOURCE_DIR}/../features/lorawan/lorastack"
-  "${PROJECT_SOURCE_DIR}/../features/lorawan/lorastack/mac"
-  "${PROJECT_SOURCE_DIR}/../features/lorawan/lorastack/phy"
-  "${PROJECT_SOURCE_DIR}/../features/lorawan/system"
+  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan"
```

```suggestion
  "${PROJECT_SOURCE_DIR}/../connectivity
  "${PROJECT_SOURCE_DIR}/../connectivity/lorawan/include/lorawan"
```
Also you need to move `lorawan_types.h` into `include/lorawan`. With those two changes, unit tests compile and pass.

---

Done